controllers/front/ProductController.php

```diff
@@ -270,7 +270,7 @@ public function initContent()
 				'last_qties' =>  (int)Configuration::get('PS_LAST_QTIES'),
 				'HOOK_EXTRA_LEFT' => Hook::exec('displayLeftColumnProduct'),
 				'HOOK_EXTRA_RIGHT' => Hook::exec('displayRightColumnProduct'),
-				'HOOK_PRODUCT_OOS' => Hook::exec('actionProductOutOfStock', array('product' => $this->product)),
+				'HOOK_PRODUCT_OOS' => Hook::exec('displayProductOutOfStock', array('product' => $this->product)),
```

Comment:
Hi,

Thank you for your contribution, but as @PrestaEdit and @PhpMadman say, you can't change the name of the hook. but you can create a new one.

Best regards